build.gradle

```diff
@@ -50,18 +47,14 @@ gitProperties {
 
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter'
+	implementation 'org.springframework.boot:spring-boot-starter-actuator'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-validation'
-	implementation 'org.springframework.cloud:spring-cloud-starter-openfeign'
 
 	implementation 'org.springdoc:springdoc-openapi-starter-webmvc-ui:2.1.0'
-	implementation 'io.github.openfeign:feign-okhttp'
 	implementation 'org.codehaus.janino:janino'
-	implementation 'org.apache.commons:commons-lang3'
-	implementation 'commons-fileupload:commons-fileupload:1.5'
 	implementation 'org.yaml:snakeyaml:2.0'
```

Comment:
This is defined to resolve some CVE?
If yes, let's move into a separate group with a start comment to introduce the dependency put just to for security reason